cvat/apps/engine/tests/test_rest_api_3D.py

```diff
@@ -208,6 +211,24 @@ def etree_to_dict(t):
 
 
 class Task3DTest(_DbTestBase):
+    def compare_annotations(self, annos_orig: list[dict], annos_imported: list[dict]):
+
+        for _type in set(AnnotationType):
+            key = f"{_type}s"
+            anns = annos_imported.data[key]
+            self.assertEqual(
+                len(annos_orig.data[key]),
+                len(annos_imported.data[key]),
```

Comment:
`annos_orig` and `annos_imported` are both supposed to be `list`s, and lists do not have a `data` member.